website/content/docs/v0.3/Guides/bootstrapping.md

```diff
@@ -89,11 +89,15 @@ Issue the following to create a single-node cluster:
 
 ```bash
 talosctl cluster create \
+  --image ghcr.io/talos-systems/talos:v0.11.5 \
   -p 69:69/udp,8081:8081/tcp \
   --workers 0 \
   --endpoint $PUBLIC_IP
```

Comment:
```suggestion
  --kubernetes-version 1.21.4 \
  -p 69:69/udp,8081:8081/tcp \
  --workers 0 \
  --endpoint $PUBLIC_IP
```

Any recent Talos version supports running Kubernetes 1.21.x, so there's no reason to pin Talos version.

Reply:
We need to pin to 1.21.x, as CAPI 0.3.x is not compatible with 1.22